src/sunsynk/sensors.py

```diff
@@ -267,7 +267,7 @@ def reg_to_value(self, regs: RegType) -> ValType:
             for bit in range(16):
                 msk = 1 << bit
                 if msk & b16:
-                    msg = f"F{bit + off + 1:02} " + faults.get(off + msk, "")
+                    msg = f"F{bit+off+1:02} " + faults.get(off + msk, "")
```

Comment:
Can you please format with ruff/ use the pre-commit hooks? (I believe that will add back these spaces)